src/components/ProviderInitializer/ProviderInitializer.tsx

```diff
@@ -120,6 +125,16 @@ export function ProviderInitializer() {
   }, [ledgerAccount, isLoggedIn, ledgerData]);
 
   async function refreshNetworkConfig() {
+    const shouldGetConfig =
+      !network.chainId ||
+      ![DEVNET_CHAIN_ID, TESTNET_CHAIN_ID, MAINNET_CHAIN_ID].includes(
```

Comment:
please place a comment here why this is needed

Reply:
this could also be resolved with suggestive variable naming (at least 50% of the explenation)